src/imitation/rewards/reward_nets.py

```diff
@@ -318,6 +324,134 @@ def forward(self, state, action, next_state, done):
         return outputs
 
 
+class NormalizedRewardNet(RewardNetWrapper):
+    """A reward net that normalizes the output of its base network."""
+
+    def __init__(
+        self,
+        base: RewardNet,
+        normalize_output_layer: Type[nn.Module],
+    ):
+        """Initialize the NormalizedRewardNet.
+
+        Args:
+            base: a base RewardNet
+            normalize_output_layer: The class to use to normalize rewards. This
+                can be any nn.Module that preserves the shape; e.g. `nn.Identity`,
+                `nn.LayerNorm`, or `networks.RunningNorm`.
+        """
+        # FIXME(yawen): by default, the reward output is squeezed to produce
```

Comment:
That's annyoing but I'm skeptical about BatchNorm for output normalization, so does not seem like a priority to fix. In particular RL training with every batch of mean zero reward is odd. I think it might still work: if the batch contains both a good and a bad action, gradients will point in the right direction. But any batch-to-batch difference is destroyed by this.